colibre/scripts/bh_luminosities.py

```diff
@@ -0,0 +1,172 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.stats as stats
+
+from swiftsimio import load
+
+import unyt
+from unyt import mh, cm, Gyr, speed_of_light
+from matplotlib.colors import LogNorm
+from matplotlib.animation import FuncAnimation
```

Comment:
```suggestion
```
These imports are not used in the script

Reply:
True, I hadn't noticed. Removed now from all scripts.